src/Symfony/Component/Validator/Tests/Constraints/IbanValidatorTest.php

```diff
@@ -173,19 +181,25 @@ public function testInvalidIbans($iban, $code)
     public function getInvalidIbans()
     {
         return array(
-            array('CH93 0076 2011 6238 5295', Iban::CHECKSUM_FAILED_ERROR),
-            array('CH930076201162385295', Iban::CHECKSUM_FAILED_ERROR),
+            array('CH93 0076 2011 6238 5295', Iban::INVALID_FORMAT_ERROR),
+            array('CH930076201162385295', Iban::INVALID_FORMAT_ERROR),
+            array('NO93 8601117 947', Iban::INVALID_FORMAT_ERROR),
+            array('CH930072011623852957', Iban::INVALID_FORMAT_ERROR),
+            array('CY170020 128 0000 0012 0052 7600', Iban::INVALID_FORMAT_ERROR),
+            array('FR76', Iban::INVALID_FORMAT_ERROR),
```

Comment:
```php
            // BLZ violation, valid length and checksum. Known issue.
            // array('NL08 TEST 0300 0652 64'),

            // Country code violation, valid length and checksum
            array('XX10 RABO 0300 0652 64', Iban::INVALID_COUNTRY_CODE_ERROR),
            array('ZZ40OJIC391330359QFOEXXQX4G5B8JZLQ2', Iban::INVALID_COUNTRY_CODE_ERROR),
```

I would like these to be added, since these are now false positives.